src/Form/FieldInput.js

```diff
@@ -86,7 +86,7 @@ class FieldInput extends React.Component {
 
     if (this.hasError()) {
       errorMsg = (
-        <p className={props.helpBlockClass}>
+        <p className={classNames(props.helpBlockClass)}>
```

Comment:
This particular instance gets its overrides and defaults from the form: https://github.com/mesosphere/reactjs-components/pull/279/files#diff-035be6cdc63453ed42b5cf6b6fcf34b8R308.
In the future it would probably be good to have the fields add their own defaults, but that is out of scope for this PR.